Apps/Contoso.UWP.Demo/Package.StoreAssociation.xml

```diff
@@ -2,7 +2,6 @@
 <StoreAssociation xmlns="http://schemas.microsoft.com/appx/2010/storeassociation">
```

Comment:
What is this change?

Reply:
Not very sure about this. I copied this file from Puppet app. Does it need to change back? @guperrot Do I also need to change it back in master?

Reply:
We should revert this change if we don't know what it is for.
No need to patch master until next release it can be done on this pr.

Reply:
Sorry, I changed wrong file last time.